apps/src/weblab/CdoBramble.js

```diff
@@ -10,12 +10,15 @@ import firehoseClient from '../lib/util/firehose';
 
 // Some temporary logging to diagnose possible loading issues.
 function tempLog(event, data = null) {
+  const dataJson = {
```

Comment:
nit: this isn't json yet :)

Reply:
Haha true, just wanted to call it something other than "data"